python/nano/src/bigdl/nano/tf/quantization.py

```diff
@@ -14,16 +14,14 @@
 # limitations under the License.
 #
 from typing import List, Optional
-from bigdl.nano.deps.neural_compressor.inc_api import QuantizationINC, tf_dataset_to_inc_dataloader
+from bigdl.nano.deps.neural_compressor.inc_api import quantize as inc_quantzie
 import tensorflow as tf
 from tensorflow.keras.metrics import Metric
 from bigdl.nano.utils.log4Error import invalidInputError
 
 
 def quantize(self,
```

Comment:
shall we make this interface consistent with `Trainer.quantize`?

Reply:
Finally we should have consistent api for pytorch and tensorflow. I opened another PR https://github.com/intel-analytics/BigDL/pull/4721 to include this.